uk_tax_23-24.py

```diff
@@ -24,6 +24,9 @@
 ni_rate_2 = 0.02
 
 # Student loan repayments
+student_loan_plan_1_threshold = 22015
```

Comment:
Maybe it makes sense to put the thresholds for all student loans in a dict? i.e.
```
student_loan_thresholds = {
  "Plan 1": 22015,
  "Plan 2": 27295,
  "Plan 4": 27660,
  "Plan 5": 25000,
  "Postgraduate": 21000
}
```